lib/authenticated-api/authenticated-api.ts

```diff
@@ -10,13 +11,18 @@ import * as lambdaNodeJs from "@aws-cdk/aws-lambda-nodejs";
 import * as path from "path";
 
 import { AuthenticatedApiProps } from "./authenticated-api-props";
+import { RouteUrlProps } from "./route-url-props";
+import { IAlarmAction } from "@aws-cdk/aws-cloudwatch";
 
 const DEFAULT_API_LATENCY_THRESHOLD = cdk.Duration.minutes(1);
 const DEFAULT_LAMBDA_DURATION_THRESHOLD = cdk.Duration.minutes(1);
 
 export class AuthenticatedApi extends cdk.Construct {
   readonly apiId: string;
   readonly httpApiId: string;
+  readonly httpApi: apigatewayv2.HttpApi;
```

Comment:
By adding this as `readonly` it is now accessible outside of the class i.e.

```
const myApi = new AuthenticatedApi(...)

myApi.httpApi.whateverMethodIWantToCall(...)
```

As we've discussed we don't want to expose the underlying httpApi outside of the class so this should defined as private readonly if you don't want to change it once set or just private if you do

```
private readonly httpApi: apigatewayv2.HttpApi
```